src/apprt/surface.zig

```diff
@@ -81,6 +81,9 @@ pub const Message = union(enum) {
     /// The terminal has reported a change in the working directory.
     pwd_change: WriteReq,
 
+    /// Bell
+    bell: void,
```

Comment:
`: void` isn't necessary for a tagged union variant without a value anymore. We should probably clean up all the `: void`s inside the apprt actions at some point

Reply:
Fixed.